tuning/data/data_processors.py

```diff
@@ -383,11 +375,128 @@ def _execute_data_handlers(
               {",".join([e.name for e in DataHandlerType])}'
         )
 
+    def _split_dataset(
+        self,
+        dataset_config: DataSetConfig,
+        loaded_dataset: Union[
+            Dataset, DatasetDict, IterableDataset, IterableDatasetDict
+        ],
+    ) -> Union[DatasetDict, IterableDatasetDict]:
+
+        seed = self.processor_config.sampling_seed
```

Comment:
I am not a big fan of piggybacking on top of `sampling_seed` 

if the `sampling_seed` is just a seed we use then you can rename DataPreProcessorConfig.sampling_seed to `DataPreProcessorConfig.seed`

Reply:
Sure before, the split support we only used the sampling seed for the interleave function.
but now as we use it for splitting too, I'll rename it!